share/spack/gitlab/cloud_pipelines/stacks/e4s-power/spack.yaml

```diff
@@ -28,6 +28,8 @@ spack:
         blas: [openblas]
         mpi: [mpich]
       variants: +mpi cuda_arch=70
+    adios2:
+      require: "@2.8:"
```

Comment:
@scottwittenburg do you know what is the `spack-esque` way of expressing this? Thanks!

Reply:
What's wrong with this?